bin/hop

```diff
@@ -305,6 +314,16 @@ if [ "$1" == "explain" ]; then
     shift 1
 fi
 
+PACKAGE_MANAGER="npm"
+if [ -f "yarn.lock" ]; then
+    PACKAGE_MANAGER="yarn"
+fi
+
+PACKAGE_MANAGER_RUN_CMD=("$PACKAGE_MANAGER")
+if [ "$PACKAGE_MANAGER" == "npm" ]; then
+    PACKAGE_MANAGER_RUN_CMD+=("run")
```

Comment:
I don't think you need the if, since I think you can run `yarn` scripts with `run` included

Reply:
Not all projects use yarn though. Projects based off the new template use npm

Reply:
Oh, you mean conditionally appending `run` to the command

Reply:
Ah, we weren't sure whether you could or not and didn't bother to test. Confirmed that you can. Also verified that pnpm also uses run so even if we ended up wanting to use that eventually it'd still be fine. Going to make this switch, thanks!